dspace/config/local.cfg.EXAMPLE

```diff
@@ -247,3 +247,8 @@ db.schema = public
 
 # Maximum size of a multipart request (i.e. max total size of all files in one request)
 #spring.servlet.multipart.max-request-size = 512MB
+
+#---------------------------------------------------------------#
+#-----------------STORAGE CONFIGURATIONS------------------------#
+#---------------------------------------------------------------#
+# Additional storage options (e.g. Amazon S3) are available in `assetstore.cfg`
```

Comment:
I'd recommend removing all these notes at the end of local.cfg.EXAMPLE (i.e. remove lines 250-254).  

Instead, copy your changes to dspace.cfg over to lines 50-52 above.  That way lines 50-52 in `local.cfg.EXAMPLE` are replaced with
```
# Assetstore configurations have moved to config/modules/assetstore.cfg
# and config/spring/api/bitstore.xml.
# Additional storage options (e.g. Amazon S3) are available in `assetstore.cfg`
# assetstore.dir = ${dspace.dir}/assetstore
```